'test_delete_instance' sample setup flakes with TimeoutError

From [this Kokoro run](https://source.cloud.google.com/results/invocations/d89bfcaf-d7af-4224-bd43-646edc5fbf3a/targets/github%2Fpython-bigtable%2Fsamples%2Finstanceadmin/tests):

```
    def test_delete_instance(capsys, dispose_of):
        dispose_of(INSTANCE)

        # Can't delete it, it doesn't exist
        instanceadmin.delete_instance(PROJECT, INSTANCE)
        out = capsys.readouterr().out
        assert "Deleting instance" in out
        assert f"Instance {INSTANCE} does not exist" in out

        # Ok, create it then
>       instanceadmin.run_instance_operations(PROJECT, INSTANCE, CLUSTER1)

test_instanceadmin.py:109:
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _
instanceadmin.py:77: in run_instance_operations
    operation.result(timeout=30)
.nox/py-3-7/lib/python3.7/site-packages/google/api_core/future/polling.py:130: in result
    self._blocking_poll(timeout=timeout, **kwargs)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _

self = <google.api_core.operation.Operation object at 0x7f6a44141510>
timeout = 30, retry = <google.api_core.retry.Retry object at 0x7f6a442cefd0>

    def _blocking_poll(self, timeout=None, retry=DEFAULT_RETRY):
        """Poll and wait for the Future to be resolved.

        Args:
            timeout (int):
                How long (in seconds) to wait for the operation to complete.
                If None, wait indefinitely.
        """
        if self._result_set:
            return

        retry_ = self._retry.with_deadline(timeout)

        try:
            kwargs = {} if retry is DEFAULT_RETRY else {"retry": retry}
            retry_(self._done_or_raise)(**kwargs)
        except exceptions.RetryError:
            raise concurrent.futures.TimeoutError(
>               "Operation did not complete within the designated " "timeout."
            )
E           concurrent.futures._base.TimeoutError: Operation did not complete within the designated timeout.

.nox/py-3-7/lib/python3.7/site-packages/google/api_core/future/polling.py:111: TimeoutError
```

--- a/samples/instanceadmin/instanceadmin.py
+++ b/samples/instanceadmin/instanceadmin.py
@@ -74,7 +74,7 @@ def run_instance_operations(project_id, instance_id, cluster_id):
         # Create instance with given options
         operation = instance.create(clusters=[cluster])
         # Ensure the operation completes.
-        operation.result(timeout=30)
+        operation.result(timeout=60)
         print("\nCreated instance: {}".format(instance_id))
     # [END bigtable_create_prod_instance]
 
@@ -159,7 +159,7 @@ def add_cluster(project_id, instance_id, cluster_id):
         else:
             operation = cluster.create()
             # Ensure the operation completes.
-            operation.result(timeout=30)
+            operation.result(timeout=60)
             print("\nCluster created: {}".format(cluster_id))
         # [END bigtable_create_cluster]
 

